src/main/scala/io/iohk/ethereum/jsonrpc/EthJsonMethodsImplicits.scala

```diff
@@ -174,6 +175,60 @@ object EthJsonMethodsImplicits extends JsonMethodsImplicits {
         value = value.getOrElse(0),
         data = data.getOrElse(ByteString("")))
     }
+  }
+
+  implicit val eth_getCode = new JsonDecoder[GetCodeRequest] with JsonEncoder[GetCodeResponse] {
+    def decodeJson(params: Option[JArray]): Either[JsonRpcError, GetCodeRequest] =
+      params match {
+        case Some(JArray((address: JString) :: (blockStr: JString) :: Nil)) =>
+          for {
+            addr <- tryExtractUnformattedData(address)
```

Comment:
Minor detail: Should we check whether the length of `addr` is correct? At least `geth` returns a `"invalid argument 0: hex string has length 42, want 40 for Address" ` in that case